srcjs/helpers/helpers-theme.js

```diff
@@ -1,6 +1,7 @@
  
 // Style for touch plugin
 const setTouchStyle = (config) => {
+    // TO DO: check whether we still need this ...
```

Comment:
Not sure we need this anymore. We can check later.

Reply:
We do, removing TO DO comment